### ISSUE TITLE
How to update an existing deployment on azure?

### ISSUE THREAD

**Issue by dprokhorov17** [Azure-Samples/langfuse-on-azure#17]
Are there any docs?

**Comment by pamelafox** [comment]
Can you be more specific? I don't understand the question.

**Comment by dprokhorov17** [comment]
@pamelafox Sure... I'll try.

So, I have a Langfuse deployment on Azure. like so:

<img width="1099" alt="image" src="https://github.com/user-attachments/assets/834d533b-32bc-4ada-8b81-5a8e5ce9f0b8">

Now, if Langfuse releases a new version, how can I update my existing deployment then?

**Comment by dprokhorov17** [comment]
@pamelafox No, I am not talking about the v3. I just wanted to know what steps I need to take in order to update my langfuse v2 azure deployment. In particular, I'm interested in the multi modal feature (https://langfuse.com/docs/tracing-features/multi-modality)

**Comment by Steffen911** [comment]
Hey @dprokhorov17 ,
The multi-modal features are only available to self-hosters on v3, i.e. you'd have to wait for the update of this sample to be V3 ready or move to a self-managed deployment.

**Comment by dprokhorov17** [comment]
Hi @Steffen911,

thanks for the response. I just discovered that the my running container was crashed:

```
Successfully Connected to container: 'langfuse-rh7o3pvcdoavq-app' [Revision: 'langfuse-rh7o3pvcdoavq-app--e60cwrz-b5dc6c66f-5gb2j', Replica: 'langfuse-rh7o3pvcdoavq-app--e60cwrz']
2024-12-10T08:06:30.554183753Z Error: CLICKHOUSE_URL is not configured. Migrating from V2? Check out migration guide at
```

so, I pinned the major version to V2.

I hope that the update of this sample to be V3 ready will not takes ages...

**Comment by pamelafox** [comment]
I talked with the Langfuse team last week, and I am deprecating this repo in favor of their v3 repo that uses Terraform to deploy to Azure:
https://github.com/langfuse/langfuse-terraform-azure

Please try migrating to that, and post in the issue tracker there if there are ways that Langfuse or Azure could make that deployment easier. Thanks!
